### PR TITLE
LRQA-77422 | master

### PR DESCRIPTION
**Pull request by chaowang-123**
https://issues.liferay.com/browse/LRQA-77422
Submit to master.
Hi,  @liferay-staging please review this, thanks!

cc @JoyceWang08211  @gaborlovasdotcom

### CODE REVIEW

**Comment by chaowang-123** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase [hunk 1]
```diff
@@ -36,6 +36,53 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPP-8122. Verify that when remote publishing, the child pages are not published when they are unselected."
+	@priority = "3"
+	test AssertCanotPublishChildPageWhenUnselected {
+		property test.name.skip.portal.instance = "RemoteStaging#AssertCanotPublishChildPageWhenUnselected";
+
+		task ("Given: two children pages for 'Staging Test Page'") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase:44]: "Given: Add two children pages for Staging Test Page"

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase [hunk 2]
```diff
@@ -36,6 +36,53 @@ definition {
...
+			for (var i : list "1,2") {
+				JSONLayout.addPublicLayout(
+					groupName = "Site Name",
+					layoutName = "Test Child Page ${i}",
+					parentLayoutName = "Staging Test Page");
+			}
+		}
+
+		task ("When: In Remote Publish settings, select only the parent Page and not the child pages and publish to remote live.") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase:53]: Summarize at the business level, no need to write down every action.
e.g. 
"When: Publish only the Parent Page and not the child pages to remote live"

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase [hunk 3]
```diff
@@ -36,6 +36,53 @@ definition {
...
+			Staging.gotoPublishToLive();
+
+			Staging.configurePublishToLive(
+				uncheckChildPages = "true",
+				uncheckChildPagesNameList = "Test Child Page 1,Test Child Page 2");
+
+			Staging.publishToLive(remoteStaging = "true");
+		}
+
+		task ("Then: Assert can't access child pages on the Remote Staged Site") {
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase:67]: "Then: Assert that child pages aren't published on the remote site"

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase [hunk 4]
```diff
@@ -36,6 +36,53 @@ definition {
...
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://localhost:9080",
+				userEmailAddress = "test@liferay.com");
+
+			Navigator.gotoSitePage(
+				baseURL = "http://localhost:9080",
+				pageName = "Staging Test Page",
+				siteName = "Remote Site");
+
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase:79]: Please ensure the parent page is published to remote site. Need to add an assertion for the parent page.

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**